### PR TITLE
Workflow test

### PR DESCRIPTION
**Pull request by Zagryzyn**

### CODE REVIEW

**Comment by Zagryzyn** [comment]
Seems it works fine with Pandapower backend. Should I try to switch OS to Windows and run on PSSE?

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.9"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pipenv
+        pipenv install --skip-lock
```
Comment (dvlazariev) [inline .github/workflows/python-app.yml:27]:
```suggestion
        pipenv install --skip-lock --dev
    - name: Check formatting
      run: |
        isort . --check
        black . --check
```

**Review by dvlazariev — APPROVED** [review]
Super!

**Comment by dmy3k** [comment]
Nice work!

**Pull request merged**